src/css/table-of-contents.scss

```diff
@@ -3,23 +3,41 @@
 .table-of-contents {
```

Comment:
The left padding here squashes longer text strings more on medium breakpoints vs the current site making it a bit hard to read at a glance

![Screenshot 2024-01-08 at 12 30 50 PM](https://github.com/cypress-io/cypress-documentation/assets/6298501/43955831-a7c2-4547-9a56-f3c19fcd9f4e)